### DISCUSSION TITLE
Collecting Requirements/Ideas/Features for v1.12.0

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [vmware-tanzu/velero#6217] [General discussions]
Hello Community!

Velero v1.11 is out and before we start working on v1.12, we’d love to brainstorm with you on ideas and requirements for the feature list of this next release!

Please review what we have already laid out in [the v1.12 roadmap](https://github.com/vmware-tanzu/velero/wiki/1.12-Roadmap) and share your thoughts/ideas/requirements/feedback here so we can all work together on building out the next version of Velero.

Please check the old discussion https://github.com/vmware-tanzu/velero/discussions/5467 if you need some inspiration :) 
Orlix

**Comment by OrlinVasilev** [comment]
@pradeepkchaturvedi

**Comment by draghuram** [comment]
The correct link to 1.12 roadmap seems to be: https://github.com/vmware-tanzu/velero/wiki/1.12-Roadmap-(Draft).

**Reply by OrlinVasilev** [reply]
that's the draft I suppose @pradeepkchaturvedi will remove the draft at some point when official!
